indico/htdocs/js/indico/modules/surveys.js

```diff
@@ -53,6 +53,62 @@
         $('.js-add-question-dropdown').dropdown();
     }
 
+    global.setupSubmissionButtons = function setupSubmissionButtons() {
+        $('.submission-ids').click(function(ev) {
+            var $this = $(this);
+            if ($this.prop('checked')) {
+                $('#export-submissions').text($T('Export selected submissions'));
+            } else {
+                var filterFn = function () {
+                    return $(this).prop('checked');
+                };
+
+                if ($('.submission-ids').filter(filterFn).length == 0) {
+                    $('#export-submissions').text($T('Export all submissions'));
+                }
+            }
+        });
+
+        $('#export-submissions').click(function(evt) {
+            var $this = $(this);
+            var submissionIds = $('.submission-ids:checked').map(function(index, obj) {
+                return $(obj).val();
+            }).get();
+
+            var form = $('<form>', {
+                method: 'POST',
+                action: $this.data('href'),
+            });
+
+            submissionIds.forEach(function(element) {
```

Comment:
`$.each()` or `_.each()` - while we don't officially support the IE version without forEach anymore, there's no real drawback in using one of the library functions instead